lib/console_web/controllers/invitation_controller.ex

```diff
@@ -14,21 +14,28 @@ defmodule ConsoleWeb.InvitationController do
   def create(conn, %{"invitation" => attrs}) do
     current_user = conn.assigns.current_user
     current_organization = conn.assigns.current_organization
-    organization = Organizations.get_organization!(current_user, attrs["organization"])
 
-    if current_organization.id == organization.id and current_user.email != attrs["email"] do
-      with {:ok, %{ invitation: invitation, user: _ }} <-
```

Comment:
using this `with` clause, we weren't catching the error when the invitation (or user) fails to be created and thus the whole thing is rolled back

Reply:
what do you mean? would Organizations.create_invitation return {:ok , _} if it fails?

Reply:
we'd get `no function clause matching` error